### ISSUE TITLE
Document infrastructure requirements needed

### ISSUE THREAD

**Issue by davetropeano** [Cloud-Schematics/LAMP#4]
From: https://bigblue.aha.io/ideas/IDEAINT-I-750

@Rachael-Graham with the LAMP (and other templates) in addition to the usual Softlayer API key the user's SL account has to have permissions to be able to create the resources. If their account does not have sufficient permission levels then terraform will spit out errors like:

```
Errors received from Schematics for not having network permissions to add a public IP address.

2017/11/10 16:00:13 Terraform apply | * ibm_compute_ssh_key.ssh_key: Error creating SSH Key: SoftLayer_Exception_PermissionDenied: You do not have permission to manage ssh keys. (HTTP 500)

2017/11/13 13:58:48 Terraform apply | * ibm_compute_vm_instance.vm: Error ordering virtual guest: SoftLayer_Exception_Order_Item_Rule: 100 Mbps Public & Private Network Uplinks must be ordered with permission Add Compute with Public Network Port. (HTTP 500)
```

We should document the account/infrastructure requirements. We may be able to use boilerplate language to the effect of *Note: Your Softlayer and/or IBM Cloud account must have sufficient permissions to be able to create the requested resources or you will get a permissions error*

**Comment by Rachael-Graham** [comment]
@davetropeano I'd like to point users to a link on how to check these permissions. I found this doc for Infrastructure permissions: https://console.bluemix.net/docs/iam/mnginfra.html#managing-infrastructure-access

Any ideas for the IBM Cloud permissions? there's a similar IAM doc: https://console.bluemix.net/docs/iam/mngiam.html#iammanidaccser

**Comment by davetropeano** [comment]
@Rachael-Graham yes, these two links seem to be sufficient. If you could recommend boilerplate language to use I'll help with editing the templates to add the language to the `README.md` file

**Comment by Rachael-Graham** [comment]
No worries I have drafts already. Will tag Kate on them & then will merge them!
